### DISCUSSION TITLE
Support for gpt-4-turbo-vision

### DISCUSSION THREAD

**Discussion by easontsai** [microsoft/autogen#2632] [General]
Hi team,

It is any plan to support gpt-4-turbo-vision?

**Comment by ekzhu** [comment]
Have you tried using it with Multi Modal agent? https://microsoft.github.io/autogen/docs/reference/agentchat/contrib/multimodal_conversable_agent

cc @BeibinLi

**Comment by easontsai** [comment]
Yes, I have. Indeed, I want to use image input and function call in the gpt-4-turbo with vision and It seems azure openai api not support yet.
